pkg/scheduler/server/internal/cron/cron.go

```diff
@@ -50,29 +53,38 @@ type Interface interface {
 	// framework and database. Blocks until Etcd and the Cron library are ready.
 	Client(ctx context.Context) (api.Interface, error)
 
-	// AddWatch adds a watch for jobs to the connection pool.
-	AddWatch(*schedulerv1pb.WatchJobsRequestInitial, schedulerv1pb.Scheduler_WatchJobsServer) error
+	// AddJobsWatch adds a watch for jobs to the connection pool.
+	AddJobsWatch(*schedulerv1pb.WatchJobsRequestInitial, schedulerv1pb.Scheduler_WatchJobsServer) error
```

Comment:
For consistency, can it be?
```suggestion
	JobsWatch(*schedulerv1pb.WatchJobsRequestInitial, schedulerv1pb.Scheduler_WatchJobsServer) error
```

Reply:
:+1: